pkg/defaultcomponents/defaults.go

```diff
@@ -68,27 +67,28 @@ func Components() (component.Factories, error) {
 		healthcheckextension.NewFactory(),
 		pprofextension.NewFactory(),
 	}
-
 	for _, ext := range factories.Extensions {
 		extensions = append(extensions, ext)
 	}
-
 	factories.Extensions, err = component.MakeExtensionFactoryMap(extensions...)
 	if err != nil {
 		errs = multierr.Append(errs, err)
 	}
 
 	// enable the selected receivers
-	factories.Receivers, err = component.MakeReceiverFactoryMap(
-		prometheusreceiver.NewFactory(),
-		otlpreceiver.NewFactory(),
```

Comment:
otlpreceiver not needed?

Reply:
It's part of the core receivers and is included as part of the for loop below.